package.json

```diff
@@ -66,7 +66,7 @@
     "test": "jest",
     "test:react-16": "USE_REACT_16=true jest",
     "test:coverage": "jest --coverage",
-    "test:website": "start-server-and-test 'yarn serve:website' http://localhost:9000 'yarn cypress run --record'",
+    "test:website": "yarn cypress run --record",
```

Comment:
Why are you keeping the --record flag?

Reply:
That's how Cypress records the test run for you to watch in the dashboard, no?